### DISCUSSION TITLE
Real custom units.

### DISCUSSION THREAD

**Discussion by firatsarlar** [tailwindlabs/tailwindcss#13954] [Ideas]
During theming, you may want to create a variable like navBarHeight. Currently, we can extend the spacing section in the config, and it works. However, if you want to use the same variable (unit value) in the leading section, you need to redefine it in the leading section again. This does not seem abstract enough and creates code repetition.

**Comment by wongjn** [comment]
You could consider [referencing other values in your theme configuration](https://tailwindcss.com/docs/theme#referencing-other-values).
